ansible-ipi-install/group_vars/all

```diff
@@ -0,0 +1,47 @@
+# The global variable file for IPI on Baremetal installation
+
+# The provisioning NIC (NIC1) used on all bm nodes
+prov_nic: eno1
+
+# The public NIC (NIC2) used on all bm nodes
+pub_nic: eno2
+
+# Activation-key for proper setup of subscription-manager
+activation_key: ""
+
+# Activation-key org_id for proper setup of subscription-manager
+org_id: ""
+
+# The directory used to store the cluster configuration files (install-config.yaml, pull-secret.txt, metal3-config.yaml)
+dir: /home/{{ myuser }}/clusterconfigs
```

Comment:
to ensure this is unique (since one provisioning host might be used for multiple clusters) would if we used the clustername in this?

something like 

```
dir: /home{{ myuser }}/clusterconfigs/{{ cluster }}
```

Reply:
Initially I was in agreement with this, but I'm going to put multiple cluster installs a feature enhancement for this playbook versus it delaying getting the playbook into this repo.